src/GridView/GridView.js

```diff
@@ -11,6 +11,19 @@ import config from './../config/config.js';
 import Header from "../Header";
 
 class GridView extends ConnectionComponent {
+	constructor() {
+		super();
+		this.getChildEntityTypes = this.getChildEntityTypes.bind(this);
+        this.getEntityType = this.getEntityType.bind(this);
+	}
+
+    getEntityType(entity, hierarchy) {
```

Comment:
As a developer I would expect this function to return an Object, not an array.
Line 41 looks quite confusing ;)